app/src/main/java/com/example/sharingang/items/ItemListener.kt

```diff
@@ -0,0 +1,6 @@
+package com.example.sharingang.items
+
+// Replace the description with its ID once the database is set
+class ItemListener(val clickListener: (item: Item) -> Unit) {

```

Comment:
Question: does this have to be outside of the view holder?

Otherwise, it might be a bit cleanser to move it inside; less surface area

Reply:
No it doesn't, actually in the codelabs they put everything inside the adapter: the ViewHolder, Listener and DiffCallback.

I thought of doing this first and then noticed that you put the ViewHolder in its own file so I thought I would also create a different one for every class.

Should I move everything inside the adpater as they did or just the Listener inside the ViewHolder? What do you think?